Makefile

```diff
@@ -27,3 +27,7 @@ kube-update: update-manifests
 
 update-manifests:
 	sed 's#\(image:\) .*#\1 $(IMAGE)#' manifests/deis-logger-fluentd-daemon.yaml > manifests/deis-logger-fluentd-daemon.tmp.yaml
+
+build-plugin:
+	rake --rakefile=/rootfs/opt/fluentd/deis-output/Rakefile build
+	cp deis-output/pkg/*.gem rootfs/
```

Comment:
This target can go away now, right? I think you're now just building the plugin _during_ the build of the Dockerfile?